source/extinst.amd-gcn-shader.grammar.json

```diff
@@ -0,0 +1,23 @@
+{
+  "revision" : 2,
+  "instructions" : [
+    {
+      "opname" : "CubeFaceIndexAMD",
+      "opcode" : 1,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "'P'" }
+      ]
+    },
+    {
+      "opname" : "CubeFaceCoordAMD",
+      "opcode" : 2,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "'P'" }
+      ]
+    },
+    {
+      "opname" : "TimeAMD",
+      "opcode" : 3
+    }
```

Comment:
That would mean that this feature is enabled by Int64.  The dependency goes the other way.  This will *require* the Int64 capability. The grammar file supports that kind of dependency only for Capability themselves.